### ISSUE TITLE
Header does not render content, footer does

### ISSUE THREAD

**Issue by andraz-kozelj** [asciidoctor/asciidoctor-pdf#1130]
Header does not render any text in any of columns. I managed to get `{page-number}` up there once but that was it. It only renders the image.

I tested everything on **footer** part and it worked, changed "footer" to "header" and the header was empty.

Using alpha.18

Tried the following:
```
running-content:
  start-at: toc
header:
  padding: [0.35in, 0, 0, -0.35in]
  height: 1in
  vertical-align: center
  font:
    size: $base_font_size * 0.9
    color: $better_color_darkblue
  recto:
    left:
      content: 'image:logo.png[width="140"]'
    right:
      content: '{doctitle}'
  verso:
    left:
      content: 'image:logo.png[width="140"]'
    right:
      content: '{doctitle}'
footer:
  height: 0.75in
  font:
    size: $base_font_size * 0.9
    color: $better_color_darkblue
  recto:
    right: 
      content: '{page-number}'
    left:
      content: '{doctitle}'
  verso:
    right: 
      content: '{page-number}'
    left: 
      content: '{doctitle}'
```

**Comment by programming-wolf** [comment]
That's already fixed, see #1087. Will be part of beta.1

**Comment by mojavelinux** [comment]
Yep, as @programming-wolf said, this is a known issue. beta.1 is on the way out the door this weekend (if not sooner).
